docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_prefect.md

```diff
@@ -8,258 +8,218 @@ import InProgress from '/docs/components/warnings/_in_progress.md'
 
 <InProgress />
 
-This guide will help you run a Great Expectations with [Prefect](https://prefect.io/)
+This guide will help you run Great Expectations with [Prefect](https://prefect.io/)
 
 ## Prerequisites
 
 <Prerequisites>
 
-- [A Great Expectations instance](/docs/guides/setup/setup_overview)
-- [A Data Connection](/docs/guides/connecting_to_your_data/connect_to_data_overview)
-- [An Expectation Suite](/docs/guides/expectations/create_expectations_overview)
-- A Prefect instance. See [Prefect Quick Start guide](https://docs.prefect.io/core/getting_started/quick-start.html)
+- None
```

Comment:
Should the bullet point for having a Prefect instance still be here under prerequisites?

Reply:
Installing Prefect and prefect-great-expectations is walked through below, so not a step required earlier.